app/views/books/show.html.erb

```diff
@@ -31,7 +31,7 @@
 </ul>
 
 <% if session[:user_id].present? %>
-  <%= link_to 'Edit Book', edit_book_path(@book) %>
-  <%= link_to 'Add a Review', new_book_review_path(@book) %>
-  <%= button_to 'Delete Book', book_path(@book), method: :delete %>
+    <%= link_to 'Edit Book', edit_book_path(@book), class: "button" %>
```

Comment:
Can you remove the two extra spaces that were added to the beginning of each line inside the `if`?